proto/streamlit/proto/NewReport.proto

```diff
@@ -73,8 +73,29 @@ message Config {
   // See config option "mapbox.token".
   string mapbox_token = 4;
 
-  // See config option "server.allowRunOnSave"
+  // See config option "server.allowRunOnSave".
   bool allow_run_on_save = 5;
+
+  // See config section "customTheme".
+  CustomThemeConfig custom_theme = 6;
+}
+
+// Custom theme configuration options. Like other config options, these are set
+// in .streamlit/config.toml.
+message CustomThemeConfig {
+  enum FontFamily {
+    SANS_SERIF = 0;
+    SERIF = 1;
+    MONO = 2;
+  }
+
+  string name = 1;
+  string primary = 2;
+  string secondary = 3;
+  string sidebar = 4;
+  string main = 5;
+  string body_text = 6;
+  FontFamily font = 7;
```

Comment:
Not sure if it would be better letting this just be a string for simplicity even though it's technically not as correct from a types perspective